browser-test/src/admin/admin_predicates.test.ts

```diff
@@ -1054,211 +1086,266 @@ test.describe('create and edit predicates', () => {
-        name: 'Screen 2',
-        questions: [{name: 'text-question'}],
-      })
-      await adminPrograms.addProgramBlockUsingSpec(programName, {
-        name: 'Screen 3',
-        questions: [{name: 'number-question-equal-to'}],
-      })
-      await adminPrograms.addProgramBlockUsingSpec(programName, {
-        name: 'Screen 4',
-        questions: [{name: 'number-question-one-of'}],
-      })
-      await adminPrograms.addProgramBlockUsingSpec(programName, {
-        name: 'Screen 5',
-        questions: [{name: 'currency-question'}],
-      })
-      await adminPrograms.addProgramBlockUsingSpec(programName, {
-        name: 'Screen 6',
-        questions: [{name: 'date-question-is-earlier-than'}],
-      })
-      await adminPrograms.addProgramBlockUsingSpec(programName, {
-        name: 'Screen 7',
-        questions: [{name: 'date-question-on-or-after'}],
-      })
-      await adminPrograms.addProgramBlockUsingSpec(programName, {
-        name: 'Screen 8',
-        questions: [{name: 'date-question-age-older-than'}],
-      })
-      await adminPrograms.addProgramBlockUsingSpec(programName, {
-        name: 'Screen 9',
-        questions: [{name: 'date-question-age-younger-than'}],
-      })
-      await adminPrograms.addProgramBlockUsingSpec(programName, {
-        name: 'Screen 10',
-        questions: [{name: 'date-question-age-between'}],
-      })
-      await adminPrograms.addProgramBlockUsingSpec(programName, {
-        name: 'Screen 11',
-        questions: [{name: 'checkbox-question'}],
-      })
 
-      // Simple string predicate
-      await adminPrograms.goToEditBlockEligibilityPredicatePage(
-        programName,
-        'Screen 1',
-      )
-      await adminPredicates.addPredicates({
-        questionName: 'name-question',
-        scalar: 'first name',
-        operator: 'is not equal to',
-        value: 'hidden',
+      await test.step('Configure screen 1', async () => {
+        await adminQuestions.addNameQuestion({questionName: 'name-question'})
+        await adminPrograms.editProgramBlockUsingSpec(programName, {
+          name: 'Screen 1',
+          questions: [{name: 'name-question'}],
+        })
+        // Simple string predicate
+        await adminPrograms.goToEditBlockEligibilityPredicatePage(
+          programName,
+          'Screen 1',
+        )
+        await adminPredicates.addPredicates({
+          questionName: 'name-question',
+          scalar: 'first name',
+          operator: 'is not equal to',
+          value: 'hidden',
+        })
       })
 
-      // Single string one of a list of strings
-      await adminPrograms.goToEditBlockEligibilityPredicatePage(
-        programName,
-        'Screen 2',
-      )
-      await adminPredicates.addPredicates({
-        questionName: 'text-question',
-        scalar: 'text',
-        operator: 'is one of',
-        value: 'blue, green',
+      await test.step('Configure screen 2', async () => {
```

Comment:
suggestion (non-blocking):  In the visibility predicates test, you added the operator to the description like this "Configure screen 2 - string equal to".  I think we should be consistent and do the same thing in the eligibility test.

Reply:
I actually think the step name in the visibility test was confusing me, so I agree with you that removing it for either test is the best way to go.

Reply:
Done!